pootle/apps/accounts/adapter.py

```diff
@@ -21,12 +22,14 @@
 
 class PootleAccountAdapter(DefaultAccountAdapter):
     """Reimplementation of DefaultAccountAdapter from allauth to change
-    ajax_response.
+    ajax_response and username validation.
 
     Differences:
       - the html key is removed for performance reasons
       - form_errors is renamed to errors
+      - Unicode usernames are allowed
     """
+    username_regex = re.compile(r'^[\w.@+-]+$', re.UNICODE)
```

Comment:
In other words. This is exactly as allauth 0.29.0 which is our pinned requirement.